src/proxyServer/index.js

```diff
@@ -33,10 +33,10 @@ function addExternalMappingsInterceptor() {
 }
 
 function initProxyServer() {
-  const { certAuthority, port } = state.get()
+  const { cert, port } = state.get()
```

Comment:
Hoxy takes the CA here, but in the other local server it takes the server CERT, this is because Hoxy rewrites the CERTS. Where as the node server we use to server Webpack, just uses the server CERT.

Reply:
Can't stress pointing this out enough, as it was a major pitfall for me.